### ISSUE TITLE
Validating async relationships does not seem to work when Using Ember.Controller

### ISSUE THREAD

**Issue by jackmatt2** [DockYard/ember-validations#305]
In an Ember.ObjectController I am able to do the following (bookingSource is {async:true})

```javascript
  validations : {
    'bookingSource.content' : {
      presence: true
    }
  }
```

When Using Ember.Controller 

```javascript
  validationModel : Ember.computed.alias('model'),

  validations : {
    'validationModel.bookingSource.content' : {
      presence: true
    }
  }
```

I get an error:
```
Error while processing route: booking.account Property set failed: object in path "errors.validationModel.bookingSource" could not be found or was destroyed. Error: Property set failed: object in path "errors.validationModel.bookingSource" could not be found or was destroyed.
```

**Comment by jackmatt2** [comment]
This has been solved by #294

**Comment by martndemus** [comment]
I am going to close this issue for now, feel free to reopen this issue if you feel like it's still relevant and we should take a look at it.
